markdownlint-cli2.js

```diff
@@ -41,9 +41,15 @@ const readConfig = (dir, name, otherwise) => {
 
 // Require a module ID with the specified directory in the path
```

Comment:
Changes here will also affect `customRules` and `outputFormatters`, please add similar test cases for them.

Reply:
Since there will be three test cases for the same general concept, maybe we should have a shared term so that the cases are related.  Any thoughts on what that term should be?  For now, I'll stick with the `pre-imported` term I used for the other test case.  It was the best I came up with, but is pretty vague.

Reply:
That works for me! Same layout for all is a good suggestion.

Reply:
These two test cases sets were structured a little differently than the one for markDownItPulgins.  That on uses an intermediate directory with the scenarios under it.  These two did not use that intermediate directory.  I stayed with the existing pattern present for each scenario.

This did result is duplicating one of the custom rules.  I can move to rules up higher in the tree if you'd rather share the rules among the tests.